seqr/utils/vcf_utils.py

```diff
@@ -0,0 +1,97 @@
+import re
+
+from collections import defaultdict
+
+from seqr.utils.middleware import ErrorsWarningsException
+from seqr.utils.file_utils import file_iter
+
+BLOCK_SIZE = 65536
+
+EXPECTED_META_FIELDS ={
+    'INFO': {
+        'AC': {
+            'Number': 'A',
+            'Type': 'Integer',
+        },
+        'AN': {
+            'Number': '1',
+            'Type': 'Integer',
+        },
+        'AF': {
+            'Number': 'A',
+            'Type': 'Float',
+        }
+    },
+    'FORMAT': {
+        'GQ': {
+            'Number': '1',
+            'Type': 'Integer',
+        },
+        'GT': {
+            'Number': '1',
+            'Type': 'String',
+        }
+    }
+}
+
+REQUIRED_HEADERS = ['#CHROM', 'POS', 'ID', 'REF', 'ALT', 'QUAL', 'FILTER', 'INFO', 'FORMAT']
+
+
+def _validate_vcf(header, meta, first_line):
```

Comment:
It's hard to read/ maintain that you validate the header and meta in one function, especially since neither depends on the other. It would be much cleaner to have explicit functions for each: `_validate_vcf_header(header)` and `_validate_vcf_meta(meta)`